lib/src/button/Button.stories.tsx

```diff
@@ -76,28 +76,24 @@ export const Chromatic = () => (
     </ExampleContainer>
     <ExampleContainer>
       <Title title="Disabled" theme="light" level={4} />
-      <DxcButton label="Primary disabled" disabled icon={iconSVG} />
+      <DxcButton label="Primary disabled" disabled icon="filled_bottom_app_bar" />
     </ExampleContainer>
     <ExampleContainer>
       <Title title="With left icon" theme="light" level={4} />
-      <DxcButton label="Primary" icon={iconSVG} />
+      <DxcButton label="Primary" icon="filled_bottom_app_bar" />
     </ExampleContainer>
     <ExampleContainer>
       <Title title="With right icon" theme="light" level={4} />
-      <DxcButton label="Primary" icon={iconSVG} iconPosition="after" />
+      <DxcButton label="Primary" icon="filled_bottom_app_bar" iconPosition="after" />
     </ExampleContainer>
     <ExampleContainer>
       <Title title="Only icon" theme="light" level={4} />
-      <DxcButton icon={iconSVG} />
+      <DxcButton icon="filled_bottom_app_bar" />
     </ExampleContainer>
     <ExampleContainer>
       <Title title="Big icon (SVG)" theme="light" level={4} />
```

Comment:
Maybe rename this title to "Big icon" as there is no longer an "(image)" one.

Another choice is to add "(SVG)" to small icon.

Reply:
done